src/applications/disability-benefits/686/components/AuthorizationMessage.jsx

```diff
@@ -0,0 +1,67 @@
+import SystemDownView from '@department-of-veterans-affairs/formation/SystemDownView';
+import React from 'react';
+import appendQuery from 'append-query';
+import PropTypes from 'prop-types';
+
+import manifest from '../../526EZ/manifest.json';
+
+const { rootUrl: increaseRootUrl } = manifest;
+import { profileStatuses } from '../helpers';
+
+const { serverError, notFound } = profileStatuses;
+const nextQuery = { next: window.location.pathname };
+const signInUrl = appendQuery('/', nextQuery);
+const verifyUrl = appendQuery('/verify', nextQuery);
+
+export default class AuthorizationMessage extends React.Component {
+
+  redirectToAuthentication = () => {
+    if (!this.props.user.login.currentlyLoggedIn) window.location.replace(signInUrl);
+    else if (!this.props.user.profile.verified) window.location.replace(verifyUrl);
+  };
+
+  render() {
+
+    const { user } = this.props;
+    if (user.profile && user.profile.status === serverError) {
+    // If va_profile is null, show a system down message.
+      return <SystemDownView messageLine1="Sorry, our system is temporarily down while we fix a few things. Please try again later."/>;
+    }
+    if (user.profile && user.profile.status === notFound) {
+    // If va_profile is "not found", show message that we cannot find the user in our system.
+      return (
+        <SystemDownView
+          messageLine1="We couldn’t find your records with that information."
+          messageLine2="Please call the Vets.gov Help Desk at 1-855-574-7286, TTY: 1-800-877-8339. We're open Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET)."/>
+      );
+    }
```

Comment:
This messaging and redirect logic is based on that of the [RequiredLoginView](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/user/authorization/components/RequiredLoginView.jsx) component.

Reply:
@U-DON we are checking that a user is verified in order to authorize them to enter the 686 application, are these the appropriate errors to check for and error messages to display? These are based on your work on the RequiredLoginView component.

Reply:
I think those errors are quite outdated and will need to be updated.

Reply:
Thanks @U-DON, I've created an [issue](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13797) to review this content.